src/components/payment/checkout/CheckoutForm.js

```diff
@@ -44,6 +45,7 @@ export const TotalPayablePrice = aliasPrice()
 export const VoucherInput = aliasInput()
 export const NameInput = aliasInput()
 export const EmailInput = aliasInput()
+export const MeetupCheckbox = aliasCheckbox()
```

Comment:
@ric9176 should we call this NewsletterCheckbox instead? Meetup is the reason we show it now, Newsletter is what it is. We might change or add other conditions to the newsletter like also if workshop for instance.